ow_lander/README.md

```diff
@@ -65,6 +67,18 @@ arguments (Careful, the bools have a capital first letter: `True`, `False`),
 then call the service.
 
 ### MoveIt Setup Assistant
+The moveit setup assistant can now be easily run using the command:
+```bash
+roslaunch ow_lander setup_assistant.launch
+``` 
+
+There are different reasons why would you may want to re-run the setup assistant:
```

Comment:
'would you may' -> 'you would'   or 'you may'

Reply:
Addressed